weaviate/collection/collection_base.py

```diff
@@ -389,3 +458,13 @@ def _delete(self, name: str) -> None:
             return
 
         UnexpectedStatusCodeException("Delete collection", response)
+
+    def schema(self) -> List[_SchemaConfig]:
```

Comment:
Instead of a list, could we have a dict with {name: config}?

Reply:
Nice idea!